regen_go_proto.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+bazel build //proto/test/...
+files=($(bazel aquery 'kind(proto, //proto/...)' | grep Outputs | grep "[.]pb[.]go" | sed 's/Outputs: \[//' | sed 's/\]//' | tr "," "\n"))
```

Comment:
Probably unnecessary, but anchor the second grep at the end of the string with `$`.

Reply:
The second grep can occur mid-string, so I'm going to leave this as-is.